hellogsm-web/src/main/java/team/themoment/hellogsm/web/domain/application/mapper/ApplicationMapper.java

```diff
@@ -0,0 +1,107 @@
+package team.themoment.hellogsm.web.domain.application.mapper;
+
+import org.mapstruct.*;
+import org.mapstruct.factory.Mappers;
+import team.themoment.hellogsm.entity.domain.application.entity.Application;
+import team.themoment.hellogsm.entity.domain.application.entity.admission.AdmissionInfo;
+import team.themoment.hellogsm.entity.domain.application.entity.grade.GedAdmissionGrade;
+import team.themoment.hellogsm.entity.domain.application.entity.grade.GraduateAdmissionGrade;
+import team.themoment.hellogsm.entity.domain.application.entity.status.AdmissionStatus;
+import team.themoment.hellogsm.entity.domain.application.enums.GraduationStatus;
+import team.themoment.hellogsm.web.domain.application.dto.domain.*;
+import team.themoment.hellogsm.web.domain.application.dto.response.SingleApplicationRes;
+
+@Mapper(
+        componentModel = "spring",
+        unmappedSourcePolicy = ReportingPolicy.ERROR,
+        unmappedTargetPolicy = ReportingPolicy.ERROR,
+        typeConversionPolicy = ReportingPolicy.WARN
+)
+public interface ApplicationMapper {
+    ApplicationMapper INSTANCE = Mappers.getMapper(ApplicationMapper.class);
+
+
+    default SingleApplicationRes CreateSingleApplicationDto(Application application) {
```

Comment:
```suggestion
    default SingleApplicationRes createSingleApplicationDto(Application application) {
```
오타 수정해주세요.

그리고 다른 메서드랑 이름이 달라서 신경쓰이긴 한데, 나중에 통일할꺼니까 지금은 넘어가도 될 것 같습니다.

Reply:
나중에 통일하면 찾기가 힘들어 지금 하는 게 좋을 것 같습니다

Reply:
수정 완료했습니다